packages/wagmi-connectors/src/ledgerHid/errorCodes.ts

```diff
@@ -1,9 +1,10 @@
-export type ErrorCodes = 27906
+export type ErrorCodes = 27906 | 28160
 
 type ErrorMessages = {
   [code in ErrorCodes]: string
 }
 
 export const ERROR_MESSAGES: ErrorMessages = {
-  [27906]: 'Ledger locked! Please unlock and make sure your Ethereum app is open.'
+  [27906]: 'Ledger locked! Please unlock and make sure your Ethereum app is open.',
+  [28160]: 'Ledger device: CLA_NOT_SUPPORTED (0x6e00). Make sure the Ethereum app is open on the device.'
```

Comment:
Can the message be more user friendly?

Reply:
I'm not sure what it could be. As I see  CLA_NOT_SUPPORTED  can happen not only for the Bitcoin app but also in some other cases when the app is not updated. I'm not sure what the error means. So I thought it makes sense to show it to the user nevertheless, so it could be easier to debug.

Reply:
Hmm maybe it’s better to leave this open until we can figure out more clearly what the error means

Reply:
Yeah makes sense. it's also not as critical as another one